index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="file.css">
+</head>
+<body> <h1 class="header">Hello Microverse</h1>
+</body>
```

Comment:
- [OPTIONAL] It is a good practice to make your code readable for other developers. Your  `<h1 class="header">Hello Microverse</h1>` could be on another line like it is below.
```
   <body> 
      <h1 class="header">Hello Microverse</h1>
   </body>
```